alexis.py

```diff
@@ -252,6 +252,24 @@ def upperToCapitalize(txtAct):
                 except:
                     print(random.choice(resconst.errorResponse))
 
+            # Show 
+            elif "show" in command:
+                try:
+                    obj = command[5:]
```

Comment:
maybe show a list of all the options if user only types "show"

Reply:
sounds good